service-worker/index.js

```diff
@@ -15,134 +15,165 @@ const API_PATTERN_REGEX = API_PATTERNS.map(createRegEx);
 const ASSET_PATTERN_REGEX = ASSET_PATTERNS.map(createRegEx);
 
 const FETCH_DATA = (event, cacheName) => {
-  let request = event.request;
-  event.respondWith(
-    caches.open(cacheName).then((cache) => {
-      return cache.match(request).then((response) => {
-        if (response) {
-          return response;
-        }
-
-        return fetch(request).then((response) => {
-          if(response.status == 200) {
-            cache.put(request, response.clone());
-          }
-          return response;
-        });
-      })
-    })
-  );
+	let request = event.request;
+	event.respondWith(
+		caches.open(cacheName).then((cache) => {
+			return cache.match(request).then((response) => {
+				if (response) {
+					console.log('SW::FETCH_DATA:: SW has response in cache for : ', request.url);
```

Comment:
**General thought** : Once testing is complete in stack and before going live, if possible, pls re-organise the log messages to log important points in execution alone, (with prefixing msg as `FS-SW::`, etc.,) and making it turn on based on flag or so (ex: via query params `sw-logs=true`, etc just like BR logs)